fre/make/gfdlfremake/schema.json

```diff
@@ -1,164 +1,201 @@
 {
```

Comment:
The schema needs to move to the schema repo.  We need to think of a new way to get the schema.